includes/Migrations/Add_Media_Source_Editor.php

```diff
@@ -38,6 +40,6 @@ class Add_Media_Source_Editor extends Add_Media_Source {
 	 * @since 1.9.0
 	 */
 	protected function get_term(): string {
-		return 'editor';
+		return Media_Source::EDITOR;
```

Comment:
Instead of importing the class like this, we should inject an instance into `Add_Media_Source`. 

So 

```
public function __construct( Media_Source_Taxonomy $media_source_taxonomy, Media_Source $media_source ) {
		$this->media_source_taxonomy = $media_source_taxonomy;
        $this->media_source = $media_source;
	}
```

So here we could do this. 

```
protected function get_term(): string {
    return $this->media_source::EDITOR;
}
```

Same goes elsewhere.

Reply:
Or better yet merge  `Media_Source` and `Media_Source_Taxonomy` classes and you could just do this. 

```
protected function get_term(): string {
    return $this->media_source_taxonomy::EDITOR;
}
```

Reply:
If we just need the static const of the enum class I don't really see the need for injecting it.

See also [my other comment](https://github.com/GoogleForCreators/web-stories-wp/pull/12864#discussion_r1054383157)